content/docs/architecture/auth.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Authentication and Authorization
+weight: 60
+---
+
+
+Fluvio uses mTLS (mutual TLS) for authentication between client and the fluvio cluster.   The user id (or principle) is extracted from the client certificate then apply to authorization policy.  Currently, Fluvio supports a simple RBAC (role based access control) authorization model but it could be extended to support more complex authorization scheme in the future. 
+
+# Cluster setup TLS
+
+First Fluvio must be running with TLS enabled. Once TLS is enabled, Both SC and SPU will accept mTLS base traffic from the client.  As part of TLS based configuration, admin can also specify authorization policy file.
```

Comment:
<s> First Fluvio must be running with TLS enabled Once TLS is enabled, Both SC and SPU will accept mTLS base traffic from the client. As part of TLS based configuration, admin can also specify authorization policy file. </s>.  

Fluvio cluster must have TLS enabled for the authorization mechanism to kick in. With TLS enabled, both SC and SPU will ensure all subsequent traffic to and from the client is encrypted via mTLS. As part of TLS configuration, the cluster administrator must also define an authorization policy file (described below).